### DISCUSSION TITLE
Mac specific keyboard shortcuts

### DISCUSSION THREAD

**Discussion by hasantahir** [flxzt/rnote#643] [Development]
As an academic, would like to say a big thank you for making this tool. On macos specific shortcuts (⌃ key replaced by ⌘ key), should the changes be made in the `rnote-ui/data/ui/shortcuts.ui` file?

Thanks

**Comment by flxzt** [comment]
I wrote a comment in issue #663 . I'll close this here so that the discussion happens in a single place.
